Build is failing with 8.4 SEMPv2 schema.

The latest Solace 8.4 SEMPv2 schema has changed the format of integers in this field such that it now expects a long. Samples need to be updated to work with new schema.



```
> Building 0% > :compileJavaaven.org/maven2/com/fasterxml/jackson/module/jackson-module-jaxb-annotations/2.5.4/jackson-module-jaxb-annotations-2.5.4.jar
Download https://repo1.maven.org/maven2/javax/inject/javax.inject/1/javax.inject-1.jar
> Building 0% > :compileJavae/travis/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.7.5/e8797b15ad6a29e2e9aefcb71033f1eb51714631/jackson-datatype-jsr310-2.7.5.jar(com/fasterxml/jackson/datatype/jsr310/JavaTimeModule.class): major version 52 is newer than 51, the highest major version supported by this compiler.
> Building 0% > :compileJavamended that the compiler be upgraded.
> Building 0% > :compileJavae/travis/build/SolaceSamples/solace-samples-semp/java/src/main/java/com/solace/samples/ManageVPN.java:83: error: method setMaxMsgSpoolUsage in class MsgVpn cannot be applied to given types;
        msgVpn.setMaxMsgSpoolUsage(1500);
              ^
  required: Long
  found: int
  reason: actual argument int cannot be converted to Long by method invocation conversion
> Building 0% > :compileJava> Building 0% > :compileJava:compileJava FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.
```

--- a/docs/_tutorials/messagevpn-with-queue-java.md
+++ b/docs/_tutorials/messagevpn-with-queue-java.md
@@ -75,7 +75,7 @@ documentation]({{ site.docs-api }}){:target="_top"}:
 MsgVpn msgVpn = new MsgVpn();
 msgVpn.setMsgVpnName(messageVpnName);
 msgVpn.setAuthenticationBasicType(AuthenticationBasicTypeEnum.INTERNAL);
-msgVpn.setMaxMsgSpoolUsage(1500);
+msgVpn.setMaxMsgSpoolUsage(1500L);
 msgVpn.setEnabled(true);
 MsgVpnResponse vpnResp = sempApiInstance.createMsgVpn(msgVpn, null);        
 ```

--- a/java/src/main/java/com/solace/samples/ManageVPN.java
+++ b/java/src/main/java/com/solace/samples/ManageVPN.java
@@ -80,7 +80,7 @@ public class ManageVPN {
         MsgVpn msgVpn = new MsgVpn();
         msgVpn.setMsgVpnName(messageVpnName);
         msgVpn.setAuthenticationBasicType(AuthenticationBasicTypeEnum.INTERNAL);
-        msgVpn.setMaxMsgSpoolUsage(1500);
+        msgVpn.setMaxMsgSpoolUsage(1500L);
         msgVpn.setEnabled(true);
         MsgVpnResponse resp = sempApiInstance.createMsgVpn(msgVpn, null);        
     }

